jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/RuntimeDataServiceImplTest.java

```diff
@@ -1210,22 +1213,15 @@ public void testGetTasksAssignedAsPotentialOwnerByStatusPagingAndFiltering() {
     
     @Test
     public void testGetTasksAssignedAsPotentialOwnerSortedByAlias() {
-        List<Long> processInstanceIds = new ArrayList<Long>();
-        for (int i = 0; i < 10; i++) {
-
-            processInstanceIds.add(processService.startProcess(deploymentUnit.getIdentifier(), "org.jbpm.writedocument"));
-        }
-
+        startProcesses(10, "org.jbpm.writedocument");
+        
         List<Status> statuses = new ArrayList<Status>();
         statuses.add(Status.Ready);
         statuses.add(Status.Reserved);
 
         QueryFilter ctx = new QueryFilter(0, 5, "Status", true);
         
         List<TaskSummary> tasks = runtimeDataService.getTasksAssignedAsPotentialOwnerByStatus("salaboy", statuses, ctx);
-        assertNotNull(tasks);
-        assertEquals(5, tasks.size());
-
```

Comment:
Same here.

Reply:
There was an already existing issue with this test. 
Basically Task table is not removed between test, which leads to inconsistencies. 
Since it is apparently not related with these changes, Gonzalo will investigate this as a separate task.